run-optimized-test.sh

```
@@ -0,0 +1,17 @@
+
+# Set optimization flags
+export MEMORY_OPTIMIZED=true
+export MAX_REPOS=1
+export MAX_BRANCH_PAGES=5
+export SKIP_DETAILED_CONTENT=true
+export SKIP_AI_ANALYSIS=true
+export CACHE_TTL=3600000
+
+# Run the test with the specified command or default to 'generate'
+COMMAND=${1:-generate}
+ARGS=${@:2}
```

Shellcheck (0.10.0)

[warning] Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.

(SC2124)